packages/datadog-plugin-redis/src/index.js

```diff
@@ -13,8 +13,14 @@ class RedisPlugin extends Plugin {
   constructor (...args) {
     super(...args)
 
+    this.configNormalized = false
+
     this.addSub('apm:redis:command:start', ({ db, command, args, connectionOptions }) => {
-      this.config = normalizeConfig(this.config)
+      if (!this.configNormalized) {
+        this.config = normalizeConfig(this.config)
```

Comment:
Why not simply normalize outside of the subscription?

Reply:
because I don't have access to the config outside the subscription

Reply:
Then I would override the configure method to normalize the config instead of doing this check on every function call.